boards/Protectli/VP66xx/v0.9.0-results.csv

```diff
@@ -0,0 +1,192 @@
+Test name,Test ID,VP66x0 VP6650 v0.9.0,VP66x0 VP6670 v0.9.0
+TPM Support (Ubuntu),TPM001.002,FAIL,PASS
+TPM Support (WIndows),TPM001.003,PASS,PASS
+Verify TPM version (firmware),TPM002.001,FAIL,PASS
+Verify TPM version (Ubuntu),TPM002.002,PASS,PASS
+Verify TPM version (Windows),TPM002.003,PASS,PASS
+Check TPM Physical Presence Interface (firmware),TPM003.001,FAIL,PASS
+Check TPM Physical Presence Interface (Ubuntu),TPM003.002,PASS,PASS
+Check TPM Physical Presence Interface (Windows),TPM003.003,PASS,PASS
+Verified boot support (Ubuntu),VBO006.002,PASS,PASS
+Booting from Slot A (Ubuntu),VBO007.002,PASS,PASS
+Recovery boot popup (firmware),VBO009.001,FAIL,PASS
+Recovery boot popup confirmation (firmware),VBO010.001,FAIL,PASS
+Recovery popup is not displayed when correctly signed firmware is flashed in RW_A,VBO011.001,FAIL,PASS
+Self-signed binary is bootable without errors,VBO012.001,FAIL,PASS
+BIOS lock support (Ubuntu),BLS001.001,FAIL,PASS
+BIOS lock support deactivation (Ubuntu),BLS002.001,FAIL,PASS
+Measured Boot support (Ubuntu),MBO001.001,FAIL,PASS
+Check Secure Boot default state (firmware),SBO001.001,PASS,PASS
+UEFI Secure Boot (Ubuntu),SBO002.001,PASS,PASS
+UEFI Secure Boot (Windows),SBO002.002,PASS,PASS
+Reset Secure Boot Keys option availability (firmware),SBO006.001,PASS,PASS
+Check Password Setup option availability and default state,PSW001.001,PASS,PASS
+Password setting mechanism correctness checking,PSW002.001,PASS,PASS
+Attempt to log in with a correct password,PSW003.001,PASS,PASS
+Attempt to log in with an incorrect password,PSW004.001,PASS,PASS
+Attempt to log in with an incorrect password 3 times,PSW005.001,PASS,PASS
+Attempt to turn off setup password functionality,PSW006.001,PASS,PASS
+Attempt to set non-compilant password,PSW007.001,PASS,PASS
+Attempt to set old password,PSW008.001,PASS,PASS
+Check if ME is statically disabled,MES001.001,PASS,PASS
+Enable USB stack (firmware),USS001.001,FAIL,FAIL
+Disable USB stack (firmware),USS002.001,FAIL,FAIL
+Enable USB Mass Storage (firmware),USS003.001,FAIL,FAIL
```

Comment:
Why it doesn't work? I didn't see anything in the release notes saying that it doesn't work.

Reply:
I don't know. There is no links to any issues on the matrix. It worked in RC1, failed in RC6 and was not tested again in RC7

Reply:
Isn't the action required obvious then? it has to be retested...

Reply:
I cannot see the test anymore in the scope, so the comment is probably not valid.